### PR TITLE
updated guidelines article from hacktoberfest official site

### PR DESCRIPTION
**Pull request by mofolactic**
Added a updated guidelines article from hacktoberfest official site, to prevent spamming.

### CODE REVIEW

**Comment by crweiner** [comment]
Hi @mofolactic! Could you please work on what Code Climate flagged? Thanks!

**Comment by mofolactic** [comment]
I am sorry to say, but, I am new to this and I don't know anything about the issue that is being created. I am still trying nevertheless.
It would be great if you would help me, this will enhance my learning.

**Comment by crweiner** [comment]
Hi @mofolactic! 

If you look at the [Code Climate flag here](https://codeclimate.com/github/crweiner/hacktoberfest-swag-list/pull/324), it says that headers need to be surrounded by blank lines. That means there just has to be a space between the header and the addition you want to make. 😄 

I don't want to fix this for you because I know you can do it. ❤️ Just move your "For more information..." sentence down one line and commit it. 👍

**Comment by mofolactic** [comment]
Bro I have tried a lot, its not getting fixed... Please help what should I do now.
Sorry, for disappointing you.

File: README.md [hunk 1]
```diff
@@ -5,10 +5,11 @@ It's that time of year again! Time to get rewarded for contributing to the Open
 This repo seeks to document all of the companies giving away swag for Hacktoberfest 2020 and how you can get your hands on some free gifts for your time and help.
 
 ## Hacktoberfest is about making meaningful contributions to open source projects. SPAM for the sake of a shirt will not be tolerated
-
+
```
Comment (crweiner) [inline README.md:8]: You have a space inserted here. 🙃 That's all! I'll remove this one blank space for you, but that is what was causing the Code Climate flag.

**Comment by mofolactic** [comment]
Thanks man!!


You have set an example that how one should treat a beginner who is trying to learn something new.
I am familiar with the issues now, and, try to keep this is in my mind as well.

Cheers!!!!

**Comment by crweiner** [comment]
No problem @mofolactic! I'm glad you're taking Hacktoberfest as a time to learn, grow, and give back to the community! That's what it's all about. 😊💜💜😊

**Pull request merged**